third_party/amd/lib/TritonAMDGPUToLLVM/SchedInstructions.cpp

```diff
@@ -194,6 +266,9 @@ struct InstructionSchedHintsRewriter
       return;
     }
 
+    if (!machineDescr)
+      schedHint.emitError("unknown target architecture detected");
```

Comment:
Should we return here if missing target?

Reply:
Do you mean I need to add `return;` statement after  `schedHint.emitError(...)`? If yes, it makes sense. Done